e3sm_diags/derivations/acme.py

```diff
@@ -133,6 +133,15 @@ def qflxconvert_units(var):
     return var
 
 
+def w_convert_q(var):
+    if var.units == "mol/mol":
+        var = var * 18.0 / 28.97  # convert to mixing ratio
```

Comment:
It's an embarrassing miss...I correct the formula. Now values of Q and H2OLNO are very close, not identical though.